src/OrchardCore.Modules/OrchardCore.Lists/GraphQL/ContainedQueryObjectType.cs

```diff
@@ -0,0 +1,16 @@
+using GraphQL.Types;
+using OrchardCore.Lists.Models;
+
+namespace OrchardCore.Lists.GraphQL
+{
+    public class ContainedQueryObjectType : ObjectGraphType<ContainedPart>
```

Comment:
I find this syntax cumbersome. The amount of work to do to just call `Field()` is important. A solution could be to expose an options object for these, and in the Startup we just call a builder that lets use invoke `Field()`. We would not need to create these classes for each part,